### ISSUE TITLE
Index language code data into readable word 

### ISSUE THREAD

**Issue by marlo-longley** [sul-dlss/vt-arclight#171]
Map the values
Blocked by #170

**Comment by jcoyne** [comment]
@marlo-longley can you say more about why we want to do this?  What user value are we trying to bring though this change?

**Comment by marlo-longley** [comment]
@jcoyne Hi Justin, we don't even have the language code in the ASpace EAD yet, so this is somewhat of a placeholder ticket.

We will want to display the languages of items, and don't want @laurensorensen to have to manually convert the data herself in her spreadsheets. We do have the language codes in the data already, however.

**Comment by marlo-longley** [comment]
For example this design has "English" in the UI, not "en_US" or whatever the code is
https://github.com/sul-dlss/vt-arclight/issues/162

**Comment by jcoyne** [comment]
@marlo-longley I don't think we necessarily need to index that to show it correctly to the user.  That's just a translation, right?

**Comment by marlo-longley** [comment]
@jcoyne we will also facet on this data so I thought it had to be indexed

**Comment by jcoyne** [comment]
@marlo-longley A good way to describe that user experience is "We want to show this value on the show page and have the user facet on it"

**Comment by jcoyne** [comment]
Is this a duplicate of https://github.com/sul-dlss/vt-arclight/issues/142 then?

**Comment by marlo-longley** [comment]
@jcoyne we can change or delete the ticket. It was created just now during a group call because we thought it would be useful.

**Comment by laurensorensen** [comment]
I mapped the existing Language codes to readable words here:
https://docs.google.com/spreadsheets/d/18CDb4KKAgcmV2ctJc-FZKMVLJUXElS6djRPfNN4q9Zk/edit#gid=0

**Comment by jcoyne** [comment]
@laurensorensen is that going to be imported into ASpace?

**Comment by marlo-longley** [comment]
@jcoyne no, we want to do this mapping at index time.

**Comment by jcoyne** [comment]
@marlo-longley I'm confused.  Is there an xpath in the EAD we can use to get this data?

**Comment by marlo-longley** [comment]
@jcoyne I'm closing this. You are right it's confusing, the data has changed.
